core/src/main/java/org/eclipse/krazo/util/ControllerUtils.java

```diff
@@ -65,7 +65,7 @@ static boolean isRequestMethod(Method method) {
             return true;
         }
         // inheritance disabled if other MVC or JAX-RS annotations found
```

Comment:
We should remove this comment as it is wrong now. The method name speaks for itself here.

Reply:
Good catch. I adjusted the comment.